src/features/chains/ChainSelectField.tsx

```diff
@@ -21,13 +21,13 @@ export function ChainSelectField({ name, label, chains, onChange, disabled }: Pr
   const [field, , helpers] = useField<ChainName>(name);
   const { setFieldValue } = useFormikContext<TransferFormValues>();
 
-  const handleChange = (newChainId: ChainName) => {
-    helpers.setValue(newChainId);
+  const handleChange = (chainName: ChainName) => {
```

Comment:
thanks! A vestige of the first version of this app which used chainId everywhere